### ISSUE TITLE
Feature request: Make option for F/O side as default

### ISSUE THREAD

**Issue by merspieler** [it0uchpods/IDG-A32X#147]
Add an option that the F/O is treated as the Pilot position (The one I get to when pressing Ctrl+V)
If option is active just on the side axis `pos * -1`

**Comment by it0uchpods** [comment]
No, there is a much easier way to do it. I'll put it on the list.

Kind Regards,
Josh

**Comment by merspieler** [comment]
thanks
